inspirehep/config.py

```diff
@@ -31,6 +31,8 @@
 from invenio_records_rest.facets import range_filter, terms_filter
 
 from inspirehep.modules.records.utils import get_detailed_template_from_record
+from .modules.records.validators.validator import InspireValidator
```

Comment:
This is not used. Please review the PR and tag as `Needs Review` once it is ready